# Dati EMG

Header header

int8[8] sample1
int8[8] sample2
